commit/report.go

```diff
@@ -110,47 +110,87 @@ func (p *Plugin) Reports(
 }
 
 func (p *Plugin) ShouldAcceptAttestedReport(
-	ctx context.Context, u uint64, r ocr3types.ReportWithInfo[[]byte],
+	ctx context.Context, seqNr uint64, r ocr3types.ReportWithInfo[[]byte],
 ) (bool, error) {
-	decodedReport, err := p.reportCodec.Decode(ctx, r.Report)
+	if isActive, err := p.checkActiveInstance(ctx); err != nil || !isActive {
+		return false, err
+	}
+
+	latestSeqNr, err := p.ccipReader.GetLatestPriceSeqNr(ctx)
```

Comment:
Probably good enough for now, but I wonder if there is a better way of getting these sequence numbers? Fetching this from chain might lead to high throughput of RPC reads, which isn't ideal.

Does OCR3 emit some logs with Sequence Number by any chance or does ChainWriter keeps some sort of information about the latest sequence number transmitted?

Reply:
I thought about this initially and didn't find another way to my knowledge. ChainWriter doesn't have this AFAIK. I think it's okay to leave it for now and see what the metrics will tell.

Reply:
Hm it seems like we don't emit the sequence number at all in `CommitReportAccepted` - @mateusz-sekara if this is critical for performance in your view we need to make onchain folks aware.

Reply:
Ah actually this is the price sequence number, might be a bit trickier getting it into CommitReportAccepted :(

Reply:
I was hoping for OCR3 events. For instance, OCR2 emits `Transmit` event with every report created (including epoch and round), not sure how it works for OCR3, might be worth investigating once this become a bottleneck